<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRDetailsDialog</class>
 <widget class="QDialog" name="CRDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Request Details</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>CR-ID: Type</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 16pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="detailsTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>